modules/xfem/test/tests/crack_tip_enrichment/tests

```diff
@@ -15,5 +20,10 @@
     unique_id = true
     heavy = true
     superlu = true
+    requirement = 'The XFEM module shall have the capability to perform crack '
+                  'tip enrichment in 3D.'
```

Comment:
The XFEM module shall have the capability to include near-tip enrichment functions in 3D mechanics simulations.